test/e2e/storage/pd.go

```diff
@@ -449,7 +449,7 @@ var _ = SIGDescribe("Pod Disks", func() {
 		// Verify that disk shows up in node 0's volumeInUse list
 		framework.ExpectNoError(waitForPDInVolumesInUse(nodeClient, diskName, host0Name, nodeStatusTimeout, true /* should exist*/))
 
-		output, err := exec.Command("gcloud", "compute", "instances", "list").CombinedOutput()
+		output, err := exec.Command("gcloud", "compute", "instances", "list", "--project="+framework.TestContext.CloudConfig.ProjectID).CombinedOutput()
```

Comment:
I wonder in general, would it be better to use the gcloud programatic APIs instead of the gcloud command line?  I know for some of the PD tests, we use the programming apis to create disks.